lumigator/backend/backend/tests/conftest.py

```diff
@@ -8,6 +8,8 @@
 from uuid import UUID
 
 import boto3
+import evaluator
+import evaluator.definition
```

Comment:
This is maybe a slightly nonstandard way to import stuff in python? Idk. I would have done something like `from evaluator import definition` or `from evaluator.definition import JOB_DEFINITION` but probably just my preference 🤷

Reply:
We need to have a qualified JOB_DEFINITION since we need to use several, but otherwise I can probably take out the `{inference,evaluator}.definition` imports.

Reply:
I checked this specific use, and for clarity purposes I'd rather leave the qualified import, but just the `import evaluator` clause.